Makefile

```diff
@@ -7,16 +7,24 @@
 
 .default:
 	@echo "Usage:"
-	@echo "    make prep  -- Prepare AWS base AMI image."
-	@echo "    make build -- Create AMI image from base image by using packer."
+	@echo "    make base-image  -- Prepare AWS base AMI image."
```

Comment:
The `make base-image` only shrink the original qcow2 disk to 8G and does not create any `base AMI image`,
and `make aws-image` uploads the shrinked RAW format qcow2 image to AWS S3 bucket and create AMI image from it.

Reply:
updated